src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Network/ApplicationGateway/Add-ArmApplicationGatewayRedirectRule.ps1

```diff
@@ -0,0 +1,73 @@
+function Add-ArmApplicationGatewayRedirectRule {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("ApplicationGateway")]
+    Param(
+        [PSTypeName("ApplicationGateway")]
+        [Parameter(Mandatory, ValueFromPipeline)]
+        $ApplicationGateway,
+        [ValidatePattern('^(\[.*\]|[a-zA-Z0-9]+[a-zA-Z0-9_\-\.]*[a-zA-Z0-9_]+)$')]
+        [string]
+        $Name = "default",
+        [string]
+        [ValidateSet("Permanent", "Found", "SeeOther", "Temporary")]
+        $RedirectType = "Permanent",
+        [Parameter(Mandatory)]
+        [string]
+        $HttpListenerName,
+        [string]
+        $TargetListenerName,
+        [string]
+        $TargetUrl,
+        [bool]
+        $IncludePath = $True,
+        [bool]
```

Comment:
Change to "switch"

Reply:
If the default is true we should rename it to ExcludeQueryString and make it a switch